meshroom/core/desc/computation.py

```diff
@@ -1,13 +1,15 @@
 import math
-from enum import Enum
+from enum import IntEnum
 
 from .attribute import ListAttribute, IntParam
 
 
-class Level(Enum):
+class Level(IntEnum):
     NONE = 0
     NORMAL = 1
     INTENSIVE = 2
+    EXTREME = 2
```

Comment:
Shouldn't this be:
```suggestion
    EXTREME = 3
```
as used in https://github.com/meshroomHub/mrSubmitters/pull/1/files#diff-9dd253882930191e431fa00981aa86f2b39630037daed18a8fc848eaf3766823R77?

Reply:
yes